llvm/include/llvm/Analysis/VectorUtils.h

```diff
@@ -195,6 +195,12 @@ static constexpr char const *MappingsAttrName = "vector-function-abi-variant";
 /// the presence of the attribute (see InjectTLIMappings).
 void getVectorVariantNames(const CallInst &CI,
                            SmallVectorImpl<std::string> &VariantMappings);
+
+/// Returns a pair of the vectorized FunctionType and the mask's position when
+/// there's one, otherwise -1.
+std::optional<std::pair<FunctionType *, int>>
```

Comment:
What's the rational for conflating these two properties given `VFInfo` already has a nice interface to query for a mask's presences and position.